src/praisonai/praisonai/cli/commands/unified.py

```
@@ -76,6 +76,163 @@ def _unregister_cleanup_handlers(original_sigint, original_sigterm):
+            # Check if aiui is available first
+            result = subprocess.run([
+                sys.executable, "-c", "import praisonaiui"
+            ], capture_output=True, text=True)
+            
+            if result.returncode != 0:
+                console.print("[red]Error: aiui package not installed.[/red]")
+                console.print("[yellow]Install with: pip install aiui[/yellow]")
+                return False
+            
+            # Run the aiui script
+            subprocess.run([sys.executable, temp_script])
```

Ruff (0.15.10)

[error] `subprocess` call: check for execution of untrusted input

(S603)